java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2118,4 +2118,44 @@ void testNansToNulls() {
       assertColumnsAreEqual(expectedDouble, resultDouble);
     }
   }
+
+  @Test
+  void testIsInteger() {
+    String[] intStrings = {"A", "nan", "Inf", "-Inf", "Infinity", "infinity", "2147483647", "2147483648", "-2147483648", "-2147483649", "NULL"};
+    String[] longStrings = {"A", "nan", "Inf", "-Inf", "Infinity", "infinity", "9223372036854775807", "9223372036854775808", "-9223372036854775808", "-9223372036854775809", "NULL"};
```

Comment:
What about numbers that look like floats?

1.0 is that an integer?  or 2.3?

Reply:
That works correctly. but when we have exponents it doesn't. I think its a bug, what do you think? Although this was closed a few days ago we can possibly reopen if we have more momentum behind it https://github.com/rapidsai/cudf/issues/5202

Reply:
I just want to be sure that we document it.  This is a boolean function that checks yes it is an int or no it is not an int.  The docs talk about parsing an int.  They don't talk about is 0.00012 going to return a true or will it return a false.  will 1.2e4 return a true or will it return a false.  That is what is relevant for the documentation for this function and for this test.

Reply:
I confused myself and documented the behavior of `to_integer` instead of `isInteger` 🤦

Reply:
Have I resolved your concern? If so can we mark this as resolved?